packages/venia-concept/src/reducers/cart.js

```diff
@@ -9,7 +9,9 @@ export const initialState = {
     details: {},
     guestCartId: null,
     shippingMethods: [],
-    totals: {}
+    totals: {},
+    addToCartActiveRequest: false,
```

Comment:
Could you, please, change the name of this property to `isAddToCartRequestInProgress` or something like this just to show, that it is a boolean variable.